src/components/Voting.jsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from "react";
+import { incrementVotes} from "../api";
+const Voting = ({votes, article_id}) => {
+    const [upVote, setUpvote] = useState("upvote");
+  const [downVote, setDownvote] = useState("downvote");
+  const [currVote, setCurrVote] = useState(votes);
+  const [upVoteClicked, setUpClicked] = useState();
+  const [downVoteClicked, setDownClicked] = useState();
+
+
+  useEffect(() => {
+}, [currVote])
+const handleUpvote = () => {
+  if (!upVoteClicked) {
+    setDownClicked(false);
+    setDownvote("downvote");
+    
+    setUpvote("upvoted");
+    setCurrVote(currVote + 1);
+  
+    setUpClicked(true);
+
+    incrementVotes(article_id, 1).catch(err => {
+        console.log(err)
+        setCurrVote(currVote-1)
+    })
+  }
+};
+
+const handleDownvote = () => {
+  if (!downVoteClicked) {
+    setUpClicked(false);
+    setUpvote("upvote");
+
+    setDownvote("downvoted");
+    setCurrVote(currVote - 1);
+  
+    setDownClicked(true);
+    incrementVotes(article_id, -1).catch(err => {
+        console.log(err)
+        setCurrVote(currVote+1)
+    })
+  }
+};
+
+    return (
+        <aside className="votes">
+        <span
+          class={upVote}
+          onClick={() => {
+            handleUpvote();
+          }}
+        >
+          <svg width="36" height="36">
+            <path d="M2 14h32L18 0 10z" fill="currentColor"></path>
```

Comment:
Currently getting an error in the console looking like the attached when I run your code - is it coming from this line? 
![image](https://user-images.githubusercontent.com/61753159/211769989-65fcdaba-c033-41ab-b465-ff37b68b201a.png)